cli/command/stack/deploy_composefile.go

```diff
@@ -150,15 +159,24 @@ func buildEnvironment(env []string) (map[string]string, error) {
 	return result, nil
 }
 
-func getConfigFile(filename string) (*composetypes.ConfigFile, error) {
-	bytes, err := ioutil.ReadFile(filename)
+func getConfigFile(filename string, stdin io.Reader) (*composetypes.ConfigFile, error) {
```

Comment:
Ah, I see; yes, we may have to refactor some things. None of these are exported functions, so I guess we can do that in a follow up.